src/theme/styles/fonts.ts

```diff
@@ -5,17 +5,25 @@ import { logDebug } from 'utils'
 export interface Fonts {
   fontDefault: string
   fontVariable: string
+  fontThin: string
+  fontLight: string
+  fontLighter: string
   fontNormal: string
   fontMedium: string
   fontBold: string
+  fontSizeDefault: string
 }
 
 const fontsVariables = {
   fontDefault: 'Inter',
   fontVariable: 'Inter var',
+  fontThin: '100',
+  fontLight: '200',
+  fontLighter: '300',
```

Comment:
Confused:
`fontLighter` is heavier than `fontLight`?

Reply:
Yeah I fully understand you being confused by it. My initial rationale was to go by Regular -> Lighter (which is also a native property value `lighter` for font-weight). But that's too confusing. Will address this in this PR like


fontLight -> fontExtraLight 
fontLighter -> fontLight

Curious on your thoughts.

Reply:
Yes, that makes more sense now :)